theforeman.org/yaml/jobs/pipeline/foreman-release-pipelines.yaml

```diff
@@ -67,6 +67,11 @@
     name: 'foreman-plugins-{version}-deb-test-pipeline'
     project-type: pipeline
     sandbox: true
+    triggers:
+      - reverse:
+          jobs:
+            - foreman-{version}-deb-pipeline
```

Comment:
this will also create triggers for `foreman-3.6-deb-pipeline` etc, which don't exist, but I think this is fine and makes the smallest change to achieve the result